examples/feature_selection/plot_rfe_digits.py

```diff
@@ -12,25 +12,37 @@
 
 """  # noqa: E501
 
+import warnings
+
 import matplotlib.pyplot as plt
 
 from sklearn.datasets import load_digits
 from sklearn.feature_selection import RFE
-from sklearn.svm import SVC
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import Pipeline
+
+warnings.filterwarnings("ignore")
 
 # Load the digits dataset
 digits = load_digits()
 X = digits.images.reshape((len(digits.images), -1))
 y = digits.target
 
-# Create the RFE object and rank each pixel
-svc = SVC(kernel="linear", C=1)
-rfe = RFE(estimator=svc, n_features_to_select=1, step=1)
-rfe.fit(X, y)
-ranking = rfe.ranking_.reshape(digits.images[0].shape)
+pipe = Pipeline(
+    [("rfe", RFE(estimator=LogisticRegression(), n_features_to_select=1, step=1))]
+)
```

Comment:
We don't need to ignore the `ConvergenceWarning`, we need to scale the data. Here I propose a `MinMaxScaler` as a slightly better option than `StandardScaler` when dealing with the digits dataset (see https://github.com/scikit-learn/scikit-learn/pull/25334#issuecomment-1470697577), but I am open to any other scaling.

```suggestion
pipe = Pipeline(
    [
        ("scaler", MinMaxScaler()),
        ("rfe", RFE(estimator=LogisticRegression(), n_features_to_select=1, step=1)),
    ]
)
```